packages/core/src/external/carequality/ihe-gateway-v2/outbound/__tests__/process-xcpd.test.ts

```diff
@@ -126,3 +129,97 @@ describe("processXCPDResponse", () => {
     expect(response.patientMatch).toBeNull();
   });
 });
+
+describe("convertIheAddressToAddress", () => {
+  it("should correct invalid streetAddressLine", () => {
+    const iheAddresses = {
+      streetAddressLine: [2831],
+      city: "San Francisco",
+      state: "CA",
+      postalCode: "94101",
+      country: "USA",
+    };
+    const result = convertIheAddressToAddress(iheAddresses);
+    expect(result).toEqual({
+      line: ["2831"],
+      city: "San Francisco",
+      country: "USA",
+      postalCode: "94101",
+      state: "CA",
+    });
+    if (result) {
+      const isValid = isValidAddress(result);
+      expect(isValid).toBeTruthy();
+    } else {
+      throw new Error("Result is null");
+    }
```

Comment:
Can we move this to be part of `expect(result).toBeTruthy()`:

```ts
const result = ...
expect(result).toBeTruthy();
if (!result) throw new Error(`Missing result`);
...
```

Why:
1. It makes it easier to check the related stuff together (object present/missing);
2. Less sub-levels/scopes/indentation

Here and below